docs/source/getting-started/authenticators-users-basics.md

```diff
@@ -25,10 +25,12 @@ If this configuration value is not set, then **all authenticated users will be a
 ```{note}
 As of JupyterHub 2.0, the full permissions of `admin_users`
 should not be required.
-Instead, you can assign [roles][] to users or groups
```

Comment:
Why was this link removed?

Reply:
It seems a commit for another doc authenticators-users-basics.md was added here, I've resolved them and it has been merged.

Reply:
This removal and the links below are still present. Maybe your fixes didn't get pushed? These changes can be reverted, the rest is great!

Reply:
can i close this PR and open another? seems I made a mistake on my local and I don't want to push that

Reply:
Yes, if that's the easiest way for you, that's fine.

You can also try rebasing by using `git reset origin/main` and then only committing the changes you want to keep.

There's no difference between opening a new PR and force-pushing to the branch used by this PR. If you do open a new PR, please close this one with a link to a new one.